subsys/net/lib/zperf/Kconfig

```diff
@@ -13,6 +13,14 @@ menuconfig NET_ZPERF
 
 if NET_ZPERF
 
+config NET_ZPERF_LEGACY_HEADER_COMPAT
+	bool "Legacy iperf UDP header format"
+	help
+	  Between iperf 2.0.10 (2017) updated the UDP header format in a
```

Comment:
Was the `Between ` word a left over from previous version? The sentence looks a bit weird but as I am not a native English speaker the above could be ok to say.

Reply:
You are right, I missed this when updating the text with the outcome of your testing. I will update tomorrow.